.vscode/settings.json

```diff
@@ -40,4 +40,4 @@
         "--config-file=backend/mypy.ini"
     ],
     "editor.formatOnSave": true
-}
+}
```

Comment:
I think an empty line should be added here. or maybe it wasn't here and you added it? github diffs for this always confuse me

Reply:
Seems like it was added by the formatter.